README.md

```diff
@@ -1,4 +1,4 @@
-# ChatGPT Citation Generator
+# ChatGPT Citations
 
 ## Overview
 The ChatGPT Citation Generator is a browser extension designed to generate BibTeX citations from chat conversations in OpenAI's ChatGPT interface. This tool aims to assist students, researchers, and anyone who needs to cite conversations or information obtained from ChatGPT in their academic or professional work.
```

Comment:
Maybe rename this to ChatGPT Citations also

Reply:
done